CMakeLists.txt

```diff
@@ -1,30 +1,43 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(lms1xx)
 
+add_compile_options(-std=c++11)
+
 # Build ROS-independent library.
 find_package(console_bridge REQUIRED)
-include_directories(include ${console_bridge_INCLUDE_DIRS})
+include_directories(include SYSTEM ${console_bridge_INCLUDE_DIRS})
```

Comment:
Alright, so this basically tells the compiler to ignore any warnings coming from headers included from those directories, as they are considered to be system packages?

I'm fine with this here, but we need to make sure we actually put there only packages that are actually not any source packages from an underlay.